README.md

```diff
@@ -0,0 +1,182 @@
+
+- ### No technical details required
+
+    A key benefit of using this liquid crystal library is that there is no need to care that much about the technical details or timings when using the display. The library happily does all of this.
+
+- ### Extendable
+
+    The library is extendable, therefore it can grow to support new LCD control devices such as I2C, SPI, Serial, 1wire, RF, CAM, etc.
+
+- ### Intuitive syntax
+
+    Liquid Crystal has a simple and intuitive syntax to handle variables and functions. The user only needs to import the library and calls its functions
+
+- ### Give back
+
+    Liquid Crystal is free for everyone. The licensed library can be copied, redistributed and used in the projects, assignments or anywhere.
+
+## Functions
+
+- LiquidCrystal()
+- begin()
+- clear()
+- home()
+- setCursor()
+- write()
+- print()
+- cursor()
+- noCursor()
+- blink()
+- noBlink()
+- display()
+- noDisplay()
+- scrollDisplayLeft()
+- scrollDisplayRight()
+- autoscroll()
+- noAutoscroll()
+- leftToRight()
+- rightToLeft()
+- createChar()
+
+## Example
+
+There are many examples implemented where this library is used. You can find other examples from [Github-Liquid-Crystal](https://github.com/arduino-libraries/LiquidCrystal/tree/master/examples) and [Arduino-Reference](https://www.arduino.cc/en/Reference/LiquidCrystal). The following are the few examples of this library.
+
+### Blink
+
+This sketch prints "Hello World!" to the LCD and makes the cursor block blink.
+
+```C++
+#include <LiquidCrystal.h>
+
+const int rs = 12, en = 11, d4 = 5, d5 = 4, d6 = 3, d7 = 2;
+LiquidCrystal lcd(rs, en, d4, d5, d6, d7);
+
+void setup() {
+  lcd.begin(16, 2);
+  lcd.print("Hello World!");
+}
+
+void loop() {
+  lcd.noBlink();
+  delay(3000);
+  lcd.blink();
+  delay(3000);
+}
+```
+
+### Display
+
+This sketch prints "Hello World!" to the LCD and uses the display() and noDisplay() functions to turn on and off the display.
+
+```C++
+#include <LiquidCrystal.h>
+const int rs = 12, en = 11, d4 = 5, d5 = 4, d6 = 3, d7 = 2;
+LiquidCrystal lcd(rs, en, d4, d5, d6, d7);
+
+void setup() {
+  lcd.begin(16, 2);
+  lcd.print("Hello World!");
+}
+
+void loop() {
+  lcd.noDisplay();
+  delay(500);
+  lcd.display();
+  delay(500);
+}
+```
+
+You can find liquid crystal tutorials from [here](https://www.arduino.cc/en/Tutorial/HelloWorld?from=Tutorial.LiquidCrystal)
+
+## Contributing
+
+If you want to contribute to this project:
+
+- Report bugs and errors
+- Ask for enhancements
+- Create issues and pull requests
+- Tell others about this library
+- Contribute new protocols
+
+Please read [CONTRIBUTING.md](https://github.com/arduino-libraries/LiquidCrystal/blob/master/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Credits
+
+The authors of this library are Arduino and Adafruit. This Library is maintained by Arduino <info@arduino.cc>.
```

Comment:
I'm not a fan of this, it's good to acknowledge the work, but the same can be obtained from Github: https://github.com/arduino-libraries/LiquidCrystal/graphs/contributors

and it's an additional section to maintain. Some of the authors were also contributing on behalf of Arduino.